docs/reference/ImageDraw.rst

```diff
@@ -254,6 +254,22 @@ Methods
     :param outline: Color to use for the outline.
     :param fill: Color to use for the fill.
 
+
+.. py:method:: ImageDraw.regular_polygon(bbox, n_sides, rotation=0, fill=None, outline=None)
+
+    Draws a regular polygon inscribed in ``bbox``,
+    with ``n_sides``, and rotation of ``rotation`` degrees
+
+    :param bbox: A bounding box which inscribes the polygon
+        (e.g. bbox = [(50, 50), (150, 150)])
+    :param n_sides: Number of sides
+        (e.g. n_sides=3 for a triangle, 6 for a hexagon, etc..)
+    :param rotation: Apply an arbitrary rotation to the polygon
+        (e.g. rotation=90, applies a 90 degree rotation)
+    :param outline: Color to use for the outline.
+    :param fill: Color to use for the fill.
```

Comment:
Put these in the same order as the signature:

```suggestion
    :param fill: Color to use for the fill.
    :param outline: Color to use for the outline.
```

(I see some of the other parameters in this file are out of order or missing, but we can fix those another time :)